includes/class-wp-query-push.php

```diff
@@ -1,230 +1,193 @@
-                $this->createTableConnections();
-                $this->createTableLogs();
+                $this->create_table_connections();
+                $this->create_table_logs();
                 update_option( 'wpquerypush_db_version', '0.1.0');
                 restore_current_blog();
             }
             return;
         }
-        error_log( 'singlesite activate detected' );
-        $this->createTableConnections();
-        $this->createTableLogs();
+        $this->create_table_connections();
+        $this->create_table_logs();
         update_option( 'wpquerypush_db_version', '0.1.0');
         return;
     }
 
     public function get_blog_ids() {
         global $wpdb;
-        $blog_ids = $wpdb->get_col( "SELECT blog_id FROM {$wpdb->blogs}; " );
+        $blog_ids = $wpdb->get_col(
+            $wpdb->prepare(
+                "SELECT blog_id FROM %s;", $wpdb->blogs
+            )
+        );
         return $blog_ids;
     }
 
     public function deactivate_plugin() {
     }
 
-    public function deleteCronJobs() {
+    public function delete_cron_jobs() {
         $timestamp = wp_next_scheduled( 'wpquerypush_cron_hook' );
         wp_unschedule_event( $timestamp, 'wpquerypush_cron_hook' );
         remove_action( 'wpquerypush_cron_hook', 'wpquerypush_cron_exec', 10, 2 );
     }
 
-    public function uninstall()
-    {
+    public function uninstall() {
         if ( is_multisite() ) {
             $blog_ids = $this->get_blog_ids();
             foreach( $blog_ids as $blog_id ) {
                 switch_to_blog( $blog_id );
-                $this->dropTableConnections();
-                $this->dropTableLogs();
-                $this->deleteCronJobs();
+                $this->drop_table_connections();
+                $this->drop_table_logs();
+                $this->delete_cron_jobs();
                 delete_option( 'wpquerypush_db_version' );
                 restore_current_blog( $blog_id );
             }
             return;
         }
-        error_log( 'singlesite uninstall detected' );
-        $this->dropTableConnections();
-        $this->dropTableLogs();
-        $this->deleteCronJobs();
+        $this->drop_table_connections();
+        $this->drop_table_logs();
+        $this->delete_cron_jobs();
         delete_option( 'wpquerypush_db_version' );
         return;
     }
 
-    private function createTable($sql)
-    {
-        include_once ABSPATH . 'wp-admin/includes/upgrade.php';
-        dbDelta($sql);
-    }
-
-    private function createTableLogs()
-    {
+    private function create_table_logs() {
         global $wpdb;
         $table_name = $wpdb->prefix . $this->TABLE_NAME_LOGS;
         $wpdb_collate = $wpdb->collate;
         $sql = "CREATE TABLE {$table_name} (
-	    id INT NOT NULL AUTO_INCREMENT,
-            ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+            id INT NOT NULL AUTO_INCREMENT,
+            date_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
@prykon changing this column name will break backward compatibility for any tables already created by deployed plugins.  I also don't think the "ts" -> to "date_time" is warranted bc "ts" means "timestamp" which is what the SQL type is referred to as "TIMESTAMP"

Reply:
Good point

Reply:
Fixed